docs/proposal/20220712-garbage-collection.md

```diff
@@ -231,17 +226,22 @@ The following sequence diagram depicts what will happen when you delete a new cl
 
 ###### Controller Changes
 
-The **reconcileDelete** of controllers for `AWSCluster` and `AWSManagedControlPlane` will be modified so that garbage collection will occur if the corresponding infra cluster object has the `awsexternalresourcegc.infrastructure.cluster.x-k8s.io` finalizer.
+The **reconcileDelete** of controllers for `AWSCluster` and `AWSManagedControlPlane` will be modified so that the garbage collecter will run if the corresponding infra cluster object has the `awsexternalresourcegc.infrastructure.cluster.x-k8s.io` finalizer.
```

Comment:
TIL that `collecter` is a valid spelling! I thought it was `collector` :smile:

Reply:
Agreed it's valid, although we refer to it elsewhere as the "garbage collection service" so will keep it as is.